src/speech.cs

```diff
@@ -16,34 +18,34 @@
 namespace Speech {
 
 	[Native]
-	[iOS (10, 0)]
+	[Mac (10,15, onlyOn64: true), iOS (10,0)]
```

Comment:
ideally don't remove/re-write attributes
more generally avoid as much _damage_ as you can to any file you edit :)
that's useful when tracking changes over time, e.g. why/who/when did `X` get added/changed ?
if the line was edited then you'll find the wrong commit(s)... and have to start again :)

It also helps when merging (branches) or reverting (commits) since it will minimize the chance of conflicts

If a file was really badly formatted, then it's better to do a separate commit/PR to fix the issues, i.e. don't mix up cosmetic and functional changes

Reply:
Another reason to avoid unnecessary changes is that they make the diffs harder to read (so it takes longer to review them, and it's easier to overlook mistakes).

A single green line (addition) is much easier to understand than a red+green remove/addition.